### PR TITLE
[test_operator][tobiko] Download default customized guest image

### PR DESCRIPTION
**Pull request by eduolivares**
With recent changes in tobiko 0.8.4, the tobiko pods will download an already customized image instead of using official ubuntu images that were customized during tobiko execution.

### CODE REVIEW

File: roles/test_operator/defaults/main.yml [hunk 1]
```diff
@@ -151,6 +151,7 @@ cifmw_test_operator_tobiko_version: master
 cifmw_test_operator_tobiko_pytest_addopts: null
 cifmw_test_operator_tobiko_prevent_create: null
 cifmw_test_operator_tobiko_num_processes: null
+cifmw_test_operator_tobiko_advanced_image_url: "https://softwarefactory-project.io/ubuntu-minimal-customized-enp3s0"
```
Comment (pablintino) [inline roles/test_operator/defaults/main.yml:154]: Care to update the README to point to what's expected from it and the associated value?

Comment (eduolivares) [reply roles/test_operator/defaults/main.yml:154]: :facepalm: in a moment

**Comment by lewisdenny** [comment]
recheck

**Review by lewisdenny — APPROVED** [review]
/approve

Please take out of draft status when you would like it to merge :)

**Pull request merged**